kubernetes/src/main/java/com/linecorp/armeria/client/kubernetes/endpoints/KubernetesEndpointGroup.java

```diff
@@ -251,8 +239,54 @@ public static KubernetesEndpointGroupBuilder builder(Config kubeConfig) {
         this.portName = portName;
         this.nodeAddressFilter = nodeAddressFilter;
         this.autoClose = autoClose;
-        watchJob(this::watchNode);
+        watchJob(this::start);
```

Comment:
Rename to safeRun?
```suggestion
        safeRun(this::start);
```

Reply:
safeRun sounds like the job is executed in the current thread. Let me rename it to `scheduleJob`.

Reply:
That sounds good. 👍